wled00/colors.cpp

```diff
@@ -91,6 +91,106 @@ void setRandomColor(byte* rgb)
   colorHStoRGB(lastRandomIndex*256,255,rgb);
 }
 
+/*  
+ *generates a random palette based on color theory
+ */
+
+CRGBPalette16 generateRandomPalette(CRGBPalette16* basepalette) 
```

Comment:
You could have used a reference here to avoid ugly pointers. 😄

Reply:
ugly to you, elegant and fast to me ;)

Reply:
What's nicer: `structPtr->var` or `structRef.var`? 😀

Reply:
oh, I did not know you can do that in C++, I usually code in C.
ok, you are right, would be better.